### PR TITLE
chore: conventional commits

### PR DESCRIPTION
**Pull request by soyuka**
| Q             | A
| ------------- | ---
| Branch?       | main for features / current stable version branch for bug fixes 
| Bug fix?      | yes/no 
| New feature?  | yes/no 
| Deprecations? | yes/no
| Tickets       | Fix #... 
| License       | MIT
| Doc PR        | api-platform/docs#...

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -18,4 +18,5 @@ Additionally:
  - Features and deprecations must be submitted against main branch.
  - Legacy code removals go to the main branch.
  - Update CHANGELOG.md file.
+ - follow the [Conventional Commits specification](https://www.conventionalcommits.org/).
```
Comment (guilliamxavier) [inline .github/PULL_REQUEST_TEMPLATE.md]:
```suggestion
 - Follow the [Conventional Commits specification](https://www.conventionalcommits.org/).
```

**Comment by dunglas** [comment]
I suggest that we simplify the table accordingly because some lines aren't necessary anymore, namely "Bug fix" and "new feature".

**Comment by soyuka** [comment]
done :)

**Review by dunglas — APPROVED** [review]

**Comment by dunglas** [comment]
Thanks!

**Pull request merged**